config/ytt/static.yaml

```diff
@@ -635,38 +635,6 @@ rules:
   - update
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  labels:
-    app: tanzu-ako-operator
-  name: ako-operator-metrics-reader
```

Comment:
are we removing the metric server as well?

Reply:
Good point, let me think about it.